### PR TITLE
Nano 6.0 with default config and syntax highlighting

### PR DESCRIPTION
**Pull request by publicarray**
Enables a default config with syntax highlighting support

**Usage:**
e.g. `env TERM=xterm-color nano /volume1/web_packages/adminer/index.php`

An alternative per user setting:

```sh
echo "include /var/packages/synocli-file/target/share/nano/*.nanorc" >> ~/.nanorc
env TERM=xterm-color nano /volume1/web_packages/adminer/index.php
```

_Linked issues:_  https://discord.com/channels/732558169863225384/732559466721181738/926808496236806165 @bokkoman

FYI More syntax support is available from: https://github.com/scopatz/nanorc

### Checklist
- [ ] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

### CODE REVIEW

File: cross/nano/Makefile [hunk 1]
```diff
@@ -12,5 +12,6 @@ COMMENT  = nano is a text editor for Unix-like computing systems or operating en
 LICENSE  = GPLv2
 
 GNU_CONFIGURE = 1
+CONFIGURE_ARGS += --prefix=$(INSTALL_PREFIX)
```
Comment (hgy59) [inline cross/nano/Makefile:15]: This line is obsolete and included with `GNU_CONFIGURE = 1`
```suggestion
```

Comment (publicarray) [reply cross/nano/Makefile:15]: Thanks I thought it might be but wasn't sure.

Comment (hgy59) [reply cross/nano/Makefile]: I recently found --prefix in cross/nodejs_v16/Makefile (where GNU_CONFIGURE must not be defined).

**Comment by publicarray** [comment]
I think this is good to merge but I don't think this warrents a new release.

**Pull request merged**